server/bin/org_migrator.py

```diff
@@ -18,32 +18,22 @@
 
 
 
-# def get_table_columns(table_name):
-#     columns = None
-#     cursor = None
-
-#     try:
-#         cursor = db.execQuery('SELECT COLUMN_NAME FROM INFORMATION_SCHEMA.COLUMNS WHERE TABLE_NAME=%s;', (table_name,))
-
-#         for row in cursor:
-#             if columns is None:
-#                 columns = []
-
-#             columns.append(row[0])
-
-#         return columns
-#     finally:
-#         if cursor is not None:
-#             cursor.close()
-
-def get_cursor_columns(cursor):
+def get_cursor_columns(db, cursor):
```

Comment:
Let's remove the imports of `os` and `sys` at the top of the file since they're not used.

Reply:
done